lucene/analysis/common/src/java/org/apache/lucene/analysis/hunspell/Stemmer.java

```diff
@@ -94,6 +94,10 @@ public Stemmer(Dictionary dictionary) {
     }
 
     List<CharsRef> list = new ArrayList<>();
+    if (length == 0) {
```

Comment:
We don't accept empty words for lookup anymore, again

Reply:
LOL empty string.